site/content/agent-app-widgets/advanced-use.md

```diff
@@ -6,44 +6,197 @@ weight: 30
 
 ## Developing your own widget
 
-If you want to build your own widget, be sure to include both the [LiveChat Boilerplate](/docs/boilerplate) and [JavaScript Widget API](#javascript-widgets-api):
+If you're building your own widget and you need to interact with the Agent App, be sure to use the [Agent App SDK](#agent-app-sdk).
 
-> Our widget SDK package is hosted on NPM. You can get it with following command:
+If you've already developed a widget, you're probably using the [Widget SDK](#widget-sdk). Keep in mind that **it is no longer actively maintained**. However, we provide backward compatibility for this SDK, so your existing widgets should still be working properly. When building new widgets, we encourage you to use the [Agent App SDK](#agent-app-sdk) instead.
 
+## Accessing LiveChat data
+
+You can leverage the OAuth2.0 authorization flow to access data from the [REST API](/docs/rest-api). Head to [Sign in with LiveChat](/docs/sign-in-with-livechat) docs for more information.
+
+## Layout and Styling
+
+If you're using React, you can use the [LiveChat Design System](https://github.com/livechat/design-system), which is a component library used directly in the LiveChat user interface.
+
+We also ship a [LiveChat Boilerplate](/docs/boilerplate) – it's a lightweight CSS stylesheet to help you lift off with creating the widget interface.
```

Comment:
I would no longer mention Boilerplate. As we have the Design System publicly available, we can deprecate Boilerplate.

Reply:
✅